manuscript/180_Web_of_objects_explained.md

```diff
@@ -10,4 +10,4 @@ In this chapter, we will explore the qualities of object-oriented design which m
 Composability... again!
 -----------------------
 
-In the two previous chapters, we followed Johnny and Benjamin in discovering the benefits of and prerequisites for composability of objects. Actually, composability is the number one quality of the design we're after. After reading Johhny and Benjamin's story, you might have some questions regarding composability. Hopefully, they are among the ones answered in the next few chapters. Ready?
+In the two previous chapters, we followed Johnny and Benjamin in discovering the benefits of and prerequisites for composability of objects. Composability is the number one quality of the design we're after. After reading Johhny and Benjamin's story, you might have some questions regarding composability. Hopefully, they are among the ones answered in the next few chapters. Ready?
```

Comment:
Emphasis comes from starting with the word "Composablity", more so than when starting with "I consider". I think it's strong enough and I'd like to suggest to leave this as is (now;).